gluten-core/src/main/scala/org/apache/gluten/component/package.scala

```diff
@@ -37,6 +37,12 @@ package object component extends Logging {
     // Output log so user could view the component loading order.
     // Call #sortedUnsafe than on #sorted to avoid unnecessary recursion.
     val components = Component.sortedUnsafe()
+    require(
+      components.nonEmpty,
+      s"No component files found in container directories named with " +
+        s"'META-INF/gluten-components' from classpath. JVM classpath value " +
+        s"is: ${System.getProperty("java.class.path")}"
```

Comment:
@zhztheplayer, is user configured `extraClassPath` covered by `java.class.path`?

Reply:
Yes, usually it is included. I am investigating on issue https://github.com/apache/incubator-gluten/issues/8462 so perhaps we shouldn't rely on `java.class.path` in future. But at his moment this error message is needed.